index.js

```diff
@@ -1,58 +1,70 @@
-let bookCollection = [];
-let bookCounter = 0;
-const inputForm = document.querySelector('#addBooks');
-const displayList = document.querySelector('#bookList');
-
-const addBooks = (titleValue, authorValue) => {
-  bookCounter += 1;
-  bookCollection = [...bookCollection, { id: bookCounter, title: titleValue, author: authorValue }];
-  localStorage.setItem('awesomeBooksCollection', JSON.stringify(bookCollection));
-};
+class BookList {
+  constructor() {
+    this.bookCollection = [];
+    this.bookCounter = 0;
+    this.inputForm = document.querySelector('#addBooks');
+    this.displayList = document.querySelector('#bookList');
+    this.inputForm.addEventListener('submit', (event) => {
+      event.preventDefault();
+      const titleValue = document.getElementById('booktitle').value;
+      const authorValue = document.getElementById('authorname').value;
+      this.addBooks(titleValue, authorValue);
+      this.inputForm.reset();
+      this.showBooksList(this.bookCollection);
+    });
+  }
 
-const showBooksList = (arr) => {
-  if (arr.length <= 0) {
-    document.getElementById('bookList').innerHTML = '<h2> List of Books </h2> <ul><li> EMPTY LIST OF BOOKS <li></ul>';
-  } else {
-    const listBook = arr.map((book) => `
-      <li> '${book.title}' by ${book.author}
-      <button type='button' id='${book.id}' class='remove-button'>Remove</button>
-      </li>`).join('');
-    displayList.innerHTML = `<h2> List of Books </h2> <ul> ${listBook} <ul>`;
+  addBooks = (titleValue, authorValue) => {
+    this.bookCounter += 1;
+    this.bookCollection = [...this.bookCollection, {
+      id: this.bookCounter, title: titleValue, author: authorValue,
+    }];
+    localStorage.setItem('awesomeBooksCollection', JSON.stringify(this.bookCollection));
   }
-};
 
-const removeBooks = (ev) => {
-  const buttonId = ev.target.id;
-  bookCollection = bookCollection.filter(
-    (y) => y !== bookCollection[bookCollection.findIndex(
-      (x) => x.id === parseInt(buttonId, 10),
-    )],
-  );
-  localStorage.setItem('awesomeBooksCollection', JSON.stringify(bookCollection));
-  showBooksList(bookCollection);
-};
+  showBooksList = (arr) => {
+    if (arr.length <= 0) {
+      document.getElementById('bookList').innerHTML = '<ul class="listbook-container"><li> EMPTY LIST OF BOOKS <li></ul>';
+    } else {
+      const listBook = arr.map((book, index) => {
+        if (index % 2 === 0) {
+          return `<li class="book-item-container-light"> '${book.title}' by ${book.author}
+                  <button type='button' id='${book.id}' class='remove-button'>Remove</button>
+                  </li>`;
+        }
+        return `<li class="book-item-container-dark"> '${book.title}' by ${book.author}
+          <button type='button' id='${book.id}' class='remove-button'>Remove</button>
+          </li>`;
+      }).join('');
+      this.displayList.innerHTML = `<ul class="listbook-container"> ${listBook} <ul>`;
+    }
+  }
```

Comment:
- [ ] you were supposed to add a navigation bar with 3links: `booklist`, `add form`, `contact info` please refer to this

![nav](https://user-images.githubusercontent.com/44745855/148208100-1b946dfe-0318-4bb1-a4fb-c2107c52fbc7.png)